docs/map.xsd

```diff
@@ -59,13 +131,31 @@
   </xs:restriction>
 </xs:simpleType>
 
-<xs:simpleType name="hexColourT">
+  <xs:simpleType name="cornerterrainT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{1,3},[0-9]{1,3},[0-9]{1,3},[0-9]{1,3}"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+<xs:simpleType name="hexColorT">
   <xs:restriction base="xs:string">
-    <xs:pattern value="[a-fA-F0-9]{6}"/>
+    <xs:pattern value="([a-fA-F0-9][a-fA-F0-9]){3}"/>
   </xs:restriction>
 </xs:simpleType>
 
-<xs:complexType name="simpleDataT">
+  <xs:simpleType name="pointsT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="(\[([0-9])+,([0-9])+\])+"/>
```

Comment:
Does this pattern actually work? Doesn't it need to account for the spaces separating the coordinate pairs? Or is that what the `[` and `]` are doing? An example value is actually `0,0 10,30 30,10`.